build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
     testCompile 'org.eclipse.jetty:jetty-server:9.4.5.v20170502'
     testCompile 'org.slf4j:slf4j-simple:' + slf4jVersion
     testCompile 'org.awaitility:awaitility-groovy:3.0.0'
+    testCompile 'com.graphql-java:java-dataloader:1.0.1'
```

Comment:
Now this is quasi controversial.  Its a test dependency to prove that one could use a batch loading strategy inside graphql via Instrumentation